dandi/tests/test_download.py

```diff
@@ -147,6 +149,69 @@ def test_download_000027_resume(
     assert digester(str(nwb)) == digests
 
 
+@mark.skipif_no_network
+@pytest.mark.parametrize(
+    "dlmode", (DownloadExisting.OVERWRITE_DIFFERENT, DownloadExisting.REFRESH)
+)
+def test_download_000027_digest(
+    tmp_path: Path, dlmode: DownloadExisting, caplog: pytest.LogCaptureFixture
+) -> None:
+    from ..support.digests import Digester
+
+    # capture logs to test whether downloads happen
+    caplog.set_level(logging.DEBUG, logger="dandi")
```

Comment:
This is already done by the `capture_all_logs` autouse fixture.  Remove these two lines.